addons/auth_oauth/res_config.py

```diff
@@ -9,9 +10,15 @@
 class base_config_settings(osv.TransientModel):
     _inherit = 'base.config.settings'
 
+    def _default_server_uri(self):
+        server_uri_gmail = self.pool.get('ir.config_parameter').get_param(self._cr, SUPERUSER_ID, 'web.base.url')
```

Comment:
self.pool is old api, self._cr is new api, SUPERUSER_ID is old api... make a choice and stick it!